SaveContext.py

```diff
@@ -765,7 +765,7 @@ def get_save_context_addresses(self):
         "Bottle with Fairy"        : 'fairy',
         "Bottle with Fish"         : 'fish',
         "Bottle with Milk"         : 'milk',
-        "Bottle with Letter"       : 'letter',
+        "Ruto\'s Letter"       : 'letter',
```

Comment:
Seems like we should standardize this to "Rutos Letter".

Reply:
Also, indentation is off in this file now

Reply:
Going with Ruto's Letter since we had received a comment in the discord we'd probably want to re-instate the apostrophes later.

Reply:
fixes pushed.